x-pack/packages/kbn-elastic-assistant/impl/assistant/chat_send/use_chat_send.tsx

```diff
@@ -208,23 +206,39 @@ export const useChatSend = ({
         setCurrentConversation(updatedConversation);
       }
     }
-    setEditingSystemPromptId(defaultSystemPromptId);
+    setCurrentSystemPromptId(defaultSystemPromptId);
   }, [
     allSystemPrompts,
     clearConversation,
     currentConversation,
     setCurrentConversation,
-    setEditingSystemPromptId,
+    setCurrentSystemPromptId,
     setSelectedPromptContexts,
     setUserPrompt,
   ]);
 
+  const handleOnChatCleared = useCallback(async () => {
+    await onChatCleared();
+    await refetchCurrentUserConversations();
+  }, [onChatCleared, refetchCurrentUserConversations]);
+
+  const handleChatSend = useCallback(
```

Comment:
`handleChatSend` previously existed in `assistant/index.tsx`. As you can see, it calls `handleSendMessage`. If you check the old `assistant/index.tsx`, later on in the components `handleChatSend` gets assigned to `handleSendMessage` (`handleSendMessage={handleChatSend}`)